pages/forgetPassword.tsx

```diff
@@ -0,0 +1,164 @@
+                                r="10"
+                                stroke="currentColor"
+                                strokeWidth="4"
+                            />
+                            <path
+                                className="opacity-75"
+                                fill="currentColor"
+                                d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
+                            />
+                        </svg>
+                    </div>
+                )}
+
+                <form onSubmit={onSubmit}>
+                    <Image
+                        src="/mysejahtera-logo.png"
+                        alt="MySejahtera logo"
+                        width={50}
+                        height={50}
+                    />
+                    <div className="text-sm">
+                        Please Enter your Email or Phone number to receive SMS or Email to reset
+                        your password
+                    </div>
+
+                    <div className="mt-4">
+                        {axiosErrorMessage.length > 0 && (
+                            <div className="bg-red-500 text-white border border-red-500 text-sm rounded-md py-1 px-1 mb-3">
+                                <span role="img" aria-label="exclaimation">
+                                    ⚠️
+                                </span>{' '}
+                                {axiosErrorMessage}
+                            </div>
+                        )}
+
+                        {axiosSuccessMessage.length > 0 && (
+                            <div className="bg-green-500 text-white border border-green-500 text-sm rounded-md py-1 px-1 mb-3">
+                                <span role="img" aria-label="tick">
+                                    ✅
+                                </span>{' '}
+                                {axiosSuccessMessage}
+                            </div>
+                        )}
+
+                        <div className="relative text-left mb-3">
+                            <label htmlFor="username" className="text-gray-700 text-sm">
+                                Phone Number or Email
+                            </label>
+                            <input
+                                type="text"
+                                id="username"
+                                className="rounded-lg border-transparent flex-1 appearance-none border border-gray-300 w-full py-2 px-4 bg-white text-gray-700 placeholder-gray-400 shadow-sm text-base focus:outline-none focus:ring-2 focus:ring-primary focus:border-transparent"
+                                placeholder={'60123456789 / ali@email.com'}
+                                {...register('username', { required: true })}
+                                value={usernameValue}
+                                onChange={(e) => setUsernameValue(e.target.value)}
```

Comment:
We shouldnt need to override these (value and onChange) as it will make `useForm` hook redudant. The reason we are using that library is to minimize unneccessary re-rendering when the input field changes. This will impact perf (however it wont be visible as our project scale is very small).

I'll leave u with these
https://react-hook-form.com/
https://react-hook-form.com/api/useform/getvalues

Reply:
changed to useForm Hook in commit https://github.com/nubpro/nakvaksin/pull/5/commits/b2ecc8969fea49ab3c4e26543cdc1189af44e9a8